src/main/java/com/suse/saltstack/netapi/results/SaltStackRunResults.java

```diff
@@ -0,0 +1,16 @@
+package com.suse.saltstack.netapi.results;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SaltStackRunResults {
+
+    @SerializedName("return")
+    private List<Map<String, String>> results;
+
+    public List<Map<String, String>> getResults() {
```

Comment:
Please add javadoc comments at least for all public methods and the class itself. I know, existing result classes might not have that, but in here there is even a need to explain the results.